prj_thingy91_nrf9160ns.conf

```diff
@@ -11,6 +11,17 @@ CONFIG_ASSERT=y
 CONFIG_REBOOT=y
 CONFIG_LOG=y
 CONFIG_LOG_IMMEDIATE=y
+# Forces a maximal log level for all modules.
+# Modules saturates their specified level if it is greater than this option,
+# otherwise they use the level specified by this option instead of their default
+# or whatever was manually set. Levels are:
+#
+# 0 OFF, logging is turned off
+# 1 ERROR, maximal level set to LOG_LEVEL_ERR
+# 2 WARNING, maximal level set to LOG_LEVEL_WRN
+# 3 INFO, maximal level set to LOG_LEVEL_INFO
+# 4 DEBUG, maximal level set to LOG_LEVEL_DBG
+CONFIG_LOG_MAX_LEVEL=3
 
```

Comment:
```CONFIG_CAT_TRACKER_LOG_LEVEL_DBG=y``` Should be added to the project configuration. This way the user will get debug messages from the cat tracker app if ```CONFIG_LOG_MAX_LEVEL``` is set to 4 (Debug). All modules in the cat tracker application src folder are limited to this config.